sdk/go/kubernetes/helm/v3/pulumiTypes.go

```diff
@@ -63,21 +63,21 @@ type FetchArgs struct {
 
 // fetchArgs is a copy of FetchArgs but without using TInput in types.
 type fetchArgs struct {
-	Version     string `pulumi:"version"`
-	CAFile      string `pulumi:"caFile"`
-	CertFile    string `pulumi:"certFile"`
-	KeyFile     string `pulumi:"keyFile"`
-	Destination string `pulumi:"destination"`
-	Keyring     string `pulumi:"keyring"`
-	Password    string `pulumi:"password"`
-	Repo        string `pulumi:"repo"`
-	UntarDir    string `pulumi:"untarDir"`
-	Username    string `pulumi:"username"`
-	Home        string `pulumi:"home"`
-	Devel       *bool  `pulumi:"devel"`
-	Prov        *bool  `pulumi:"prov"`
-	Untar       *bool  `pulumi:"untar"`
-	Verify      *bool  `pulumi:"verify"`
+	Version     string `json:"version,omitempty" pulumi:"version"`
+	CAFile      string `json:"ca_file,omitempty" pulumi:"caFile"`
```

Comment:
Curious on the json tags: does snake casing here match with what is expected in helm? (as opposed to camelCase or others)

Reply:
This matches the JSON structure expected by the invoked function. It's not used by Helm directly; it's just the way I defined the interface rather than passing a huge bag of args.